framework/src/media/audio/resample/samplerate.c

```diff
@@ -183,223 +201,208 @@ static int16_t clip(int32_t x)
 }
 
 /**
- * @brief   It handles samplerate up scaling in ratio 2.0/3.0... without fraction.
- * @remarks ratio 2.0/3.0..., e.g. 16K->32K, 16K->48K, 22.05K->44.1K...
- * @param   src: pointer to resampler object.
- * @param   num_frames_out: number of frames will be stored in output buffer.
- * @param   quoti: quotient of target samplerate being divided by original samplerate.
- * @return  void
- * @see     upresample_frac()
+ * It handles sample rate up scaling in all ratio cases (i.e. inverse ratio 0.*)
+ * and sample rate down scaling cases in inverse ratio 1.* and 2.* with fraction.
  */
-static void upresample_int(resampler_t *src, int32_t num_frames_out, int32_t quoti)
+static int32_t resample_frac(src_context_t *src, int32_t *num_frames_in)
```

Comment:
What frac means exactly? With a good name of api, user can estimate role of api after see name of method ...

Reply:
`frac` means fraction, resample_frac method is used to hanlde resampling in ratio cases with fraction.
i.e. inverse ratio (original rate / desired rate) 0.*, 1.* and 2.*
and `int` means integer,  downresample_int method is used to handle resampling in ratio cases without fraction.
i.e. inverse ratio 2.0 and 3.0